.maintain/chaostest/src/commands/singlenodeheight/singlenodeheight.js

```diff
@@ -0,0 +1,135 @@
+const {Command, flags} = require('@oclif/command')
+
+class SingleNodeHeightCommand extends Command {
+  async run() {
+    const {flags} = this.parse(SingleNodeHeightCommand)
+    const imageTag = flags.image || 'parity/substrate:latest'
+    const port = flags.port || 9933
+    const url = flags.url || 'http://localhost'
+    const wait = flags.wait || 600 * 1000
+    const namespace = flags.namespace || 'substrate-ci'
```

Comment:
This would need to use a separate namespace for each test in order to run in parallel.

Reply:
here is an issue with gitlab k8s integration, does not seem like it can be configured with a master token which has project level control, unless we initiated from a project service account, seems too broad to me. what do you think? considering maybe using label as selector. (namespace would be very nice)

Reply:
I am not quite sure I follow. Why is this specific to Gitlab? As far as I know Gitlab only needs to inject the credentials via environment variables.

Cleaning up left-over resources is a lot easier using namespaces. In addition there is no additional _namespacing_ needed to enable concurrent test execution. Thus I am very much in favor of a single namespace per test.